great_expectations_cloud/agent/event_handler.py

```diff
@@ -72,6 +72,16 @@ def handle_event(self, event: Event, id: str) -> ActionResult:
         action_result = action.run(event=event, id=id)
         return action_result
 
+    @classmethod
+    def parse_event_from(cls, msg_body: bytes) -> Event:
+        try:
+            event: Event = pydantic.parse_raw_as(Event, msg_body)
+            return event
+        except (pydantic.ValidationError, JSONDecodeError):
+            # Log as bytes
+            LOGGER.error("Unable to parse event type", extra={"msg_body": f"{msg_body!r}"})
```

Comment:
```suggestion
            LOGGER.warning("Unable to parse event type", extra={"msg_body": f"{msg_body!r}"})
```
We might want this one to be a `warning` level log.
This is expected and doesn't necessarily indicate an unrecoverable error.
https://stackoverflow.com/a/2031209/6304433